### ISSUE TITLE
use-t-well is warning about use of t.timeout

### ISSUE THREAD

**Issue by gajus** [avajs/eslint-plugin-ava#291]
```diff
test('terminates HTTP server with no connections', async (t) => {
-  t.timeout(100);

  const httpServer = await createHttpServer();

  t.true(httpServer.server.listening);

  const terminator = createHttpTerminator(httpServer.server);

  await terminator.terminate();

  t.false(httpServer.server.listening);
});

```

In this code I am getting `use-t-well` error on the highlighted line.

As far as I can tell (as per https://github.com/avajs/ava/blob/master/docs/07-test-timeouts.md), this is correct usage of `t.timeout`.

**Comment by novemberborn** [comment]
Looks like it's missing from this map: https://github.com/avajs/eslint-plugin-ava/blob/1e538172f3fb7ae2a8b792db81d3b6b50b1d3386/util.js#L100

**Comment by novemberborn** [comment]
Thanks to @pvdlg for the fix. Out as `10.0.1`.
